test-node.bash

```diff
@@ -293,108 +296,123 @@ if $force_init; then
+    docker compose run --entrypoint sh sequencer -c "jq [.[]] /config/deployed_chain_info.json > /config/l2_chain_info.json"
 
     if $simple; then
         echo == Writing configs
-        docker-compose run scripts write-config --simple
+        docker compose run scripts write-config --simple
     else
         echo == Writing configs
-        docker-compose run scripts write-config
+        docker compose run scripts write-config
 
         echo == Initializing redis
-        docker-compose up --wait redis
-        docker-compose run scripts redis-init --redundancy $redundantsequencers
+        docker compose up --wait redis
+        docker compose run scripts redis-init --redundancy $redundantsequencers
     fi
 
     echo == Funding l2 funnel and dev key
-    docker-compose up --wait $INITIAL_SEQ_NODES
-    docker-compose run scripts bridge-funds --ethamount 100000 --wait
-    docker-compose run scripts bridge-funds --ethamount 1000 --wait --from "key_0x$devprivkey"
+    docker compose up --wait $INITIAL_SEQ_NODES
+    docker compose run scripts bridge-funds --ethamount 100000 --wait
+    docker compose run scripts bridge-funds --ethamount 1000 --wait --from "key_0x$devprivkey"
 
     if $tokenbridge; then
-        echo == Deploying token bridge
-        docker-compose run -e ARB_KEY=$devprivkey -e ETH_KEY=$devprivkey tokenbridge gen:network
-        docker-compose run --entrypoint sh tokenbridge -c "cat localNetwork.json"
+        echo == Deploying L1-L2 token bridge
+        rollupAddress=`docker compose run --entrypoint sh poster -c "jq -r '.[0].rollup.rollup' /config/deployed_chain_info.json | tail -n 1 | tr -d '\r\n'"`
+        docker compose run -e ROLLUP_OWNER=$sequenceraddress -e ROLLUP_ADDRESS=$rollupAddress -e PARENT_KEY=$devprivkey -e PARENT_RPC=http://geth:8545 -e CHILD_KEY=$devprivkey -e CHILD_RPC=http://sequencer:8547 tokenbridge deploy:local:token-bridge
+        docker compose run --entrypoint sh tokenbridge -c "cat network.json"
         echo
     fi
 
     if $l3node; then
         echo == Funding l3 users
-        docker-compose run scripts send-l2 --ethamount 1000 --to l3owner --wait
-        docker-compose run scripts send-l2 --ethamount 1000 --to l3sequencer --wait
+        docker compose run scripts send-l2 --ethamount 1000 --to l3owner --wait
+        docker compose run scripts send-l2 --ethamount 1000 --to l3sequencer --wait
 
         echo == Funding l2 deployers
-        docker-compose run scripts send-l2 --ethamount 100 --to user_token_bridge_deployer --wait
-        docker-compose run scripts send-l2 --ethamount 100 --to user_fee_token_deployer --wait
+        docker compose run scripts send-l2 --ethamount 100 --to user_token_bridge_deployer --wait
+        docker compose run scripts send-l2 --ethamount 100 --to user_fee_token_deployer --wait
```

Comment:
let's add to scripts/accounts.ts, under "Valid account names" - a description of:
* user_token_bridge_deployer
* user_fee_token_deployer
* l3owner
* l3sequencer